tests/test_datadog.py

```diff
@@ -42,3 +44,55 @@ def test_api_key():
     configured_api_key = DataDog.settings[DataDog.KEY_DATADOG_API_KEY]
 
     assert configured_api_key == 'test_api_key'
+
+
+def test_metrics():
```

Comment:
Rather than having all the metrics in a single test method, I'd prefer it if they were split up into functions with more descriptive names, that makes the reporting of failure much easier to understand.